ia-guide/ia-static.md

```diff
@@ -0,0 +1,345 @@
+Have a read through the information for each template you want to use, and work out how much time you are going to need to work on it.
+
+Then you can split up docs work like this:
+
+* Planning - 20%
+* Drafting - 50%
+* Reviewing and editing - 20%
+* Production - 10%
+
+So, if you want to write a How To, and you have five months to do it, you can work out how much time to spend on each task like this:
+
+Three months = Around 100 working days, not counting weekends, and allowing for a few holidays.
+You can adjust this however you like, if you're only working on weekends, for example, or if you have other projects to work on as well.
+
+* Planning - 20 days (4 weeks)
+* Drafting - 50 days (10 weeks)
+* Review & edit - 20 days (4 weeks)
+* Production - 10 days (2 weeks)
+
+There's just one more thing to do.
+
+## Prepare to adjust
+
+The most important thing you need to do is to be flexible.
+Things don't always work out, and that's OK!
+
+Perhaps you get a new job and end up with no time to work on things.
+If you've written down your plan, and spoken to others in your project about what you were going to do, others can pick up the work, and when you're able to, you can come back to it.
+
+Perhaps you get an injection of cash and can hire some writers.
+Go back to the critical paths, expand your scope, and hand that planning to them, so that they have somewhere to start, and understand your vision for the docs.
+
+Or maybe things just don't work out, and the project ends up being abandoned entirely.
+That's sad, but it's OK.
+At least you've finished up with some valuable knowledge, so you'll be even better for your next idea.
+
+So, are you ready to get started?
+
+Go [check out the templates](https://github.com/thegooddocsproject/templates/blob/master/README.md)!
+
+## Further Reading
+
+This document was largely based on the brilliant book by Abby Covert called [How to Make Sense of Any Mess](http://www.howtomakesenseofanymess.com/).
+It's a quick read, and it doesn't just apply to technical documentation, you can use the principles in that book to help you organize anything at all.
+
+If you want to delve further into information architecture for technical documentation, the most important text you need is the O'Reilly book [Information Architecture](https://www.oreilly.com/library/view/information-architecture-4th/9781491913529/).
+It has everything you need to know, including answers to questions you didn't know you needed to ask.
+It has a polar bear on the cover.
+
+![A reclining smiling banana with red writing saying "lets go banana](images/banana.svg "Lets go banana")
+
+## That's it!
+
+You've reached the end of the static Information Architecture Guide.
+If you're ready to get started, go [check out the templates](https://github.com/thegooddocsproject/templates/blob/master/README.md)
+
+If you want to try the interactive version of this guide, visit the [interactive IA Guide](don't click here the link does not exist yet).
```

Comment:
@Loquacity This is the last link you need to update -- `ia-guide/ia-cyoa.html`

Reply:
Oh, I missed one! Thanks :+1: